packages/projection-typeorm/src/operators/storeStakePoolMetadataJob.ts

```diff
@@ -20,7 +20,13 @@ export const storeStakePoolMetadataJob = typeormOperator<Mappers.WithStakePools
         })
       );
     for (const task of tasks) {
-      await pgBoss.send(STAKE_POOL_METADATA_QUEUE, task, { slot: header.slot });
+      await pgBoss.send(STAKE_POOL_METADATA_QUEUE, task, {
+        expireInHours: 48,
```

Comment:
We need to make sure that it's not possible for a job to get lost without leaving some sort of record in the database. `expireInHours` might do that if the job is not picked up?

Also, maybe we should only add `retryLimit` when we have a metadata job failure table.

Reply:
The reason for `expireInHours: 48` is to handle cases like server down.
It could be that the maintainer of the server will requires hours to restart it.

Reply:
Are you sure it will always be restarted in 48 hours? I think this job just shouldn't expire at all.

Reply:
Ah ok! If we provide no value, the default, is to expire in 15 minutes. I'll check which value to pass to give never expire.

Reply:
I wasn't able to find in doc a never expire option: c24bf1d1c sets to 1 year